commons/types/Availability.d.ts

```diff
@@ -5,16 +5,21 @@ declare namespace Availability {
     slot_size: 15 | 30 | 60;
     start_date: Date;
     dates_to_show: number;
-    available_times: TimeSlot[];
     show_ticks: boolean;
+    calendars: Calendar[];
   }
 
+  interface Calendar {
+    email_address: string;
+    availability: "free" | "busy";
+    timeslots: TimeSlot[];
+  }
   interface TimeSlot {
     start_time: Date;
     end_time: Date;
```

Comment:
Should these be `startTime` and `endTime` instead? IIRC we only want props directly exposed by the component to be snake case.

Reply:
Good idea; let's make this a task post-PR, though, as there are some other items in-flight where renaming `start_time` to `startTime` throughout the component will lead to some pretty major conflicts ( cc @heathergh  )